### DISCUSSION TITLE
Collaborative editing

### DISCUSSION THREAD

**Discussion by andreasotto** [awcodes/filament-tiptap-editor#278] [General]
I would like to add collaborative editing. I have found the following information on tiptap: https://tiptap.dev/docs/editor/guide/collaborative-editing

I have not yet understood where I could start here in this plugin. Maybe someone can give me an idea? I would also offer compensation for support to help me add collaborative editing.

As far as I understand, there are two approaches (with WebRTC and via Websockets). I am open to both. For websockets, I would prefer an integration with Soketi because I have already used that a lot in the Laravel Docker setup.

I am curious to see if there is anyone who has an idea how this could be realized.

Andreas

**Comment by awcodes** [comment]
As far as the plugin is concerned, it would just be a matter of implementing the tiptap extension as described in the README.md

I'm not sure how well it would work though considering there could be multiple editors on the same page.

Main reason it was implemented in the plugin was because it's going to get tricky having to worry about authorization too.
